instructionAPI/src/decoder/riscv/opcode_xlat.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * See the dyninst/COPYRIGHT file for copyright information.
+ *
+ * We provide the Paradyn Tools (below described as "Paradyn")
+ * on an AS IS basis, and do not warrant its validity or performance.
+ * We reserve the right to update, modify, or discontinue this
+ * software at any time.  We shall have no obligation to supply such
+ * updates or modifications or any other form of support to you.
+ *
+ * By your use of Paradyn, you understand and agree that we (or any
+ * other person or entity with proprietary rights in Paradyn) are
+ * under no obligation to provide either maintenance services,
+ * update services, notices of latent defects, or correction of
+ * defects for Paradyn.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#ifndef INSTRUCTIONAPI_RISCV_OPCODE_XLAT_H
+#define INSTRUCTIONAPI_RISCV_OPCODE_XLAT_H
+
+#include "capstone/riscv.h"
+#include "entryIDs.h"
+
+namespace Dyninst {
+namespace InstructionAPI {
+namespace riscv {
+
+  riscv_insn get_capstone_uncompressed_opcode(riscv_insn id);
+  std::string capstone_opcode_to_mnemonic(riscv_insn id);
+  entryID translate_opcode(riscv_insn id);
```

Comment:
Let's use a descriptive function name here also:

```suggestion
  entryID capstone_opcode_to_entryid(riscv_insn id);
```